pkg/lightning/backend/local/local.go

```diff
@@ -1364,63 +1367,104 @@ func (local *Backend) doImport(
 	regionSplitSize, regionSplitKeyCnt int64,
 ) error {
 	/*
-	   [prepareAndSendJob]-----jobToWorkerCh--->[workers]
-	                        ^                       |
-	                        |                jobFromWorkerCh
-	                        |                       |
-	                        |                       v
-	               [regionJobRetryer]<--[dispatchJobGoroutine]-->done
+	 [prepareAndSendJob]---jobToWorkerCh->[storeBalancer(optional)]->[workers]
+	                     ^                                             |
+	                     |                                             v
+	                     |                           [storeBalancer(optional)]
+	                     |                                             |
+	                     |                                     jobFromWorkerCh
+	                     |                                             |
+	                     |                                             v
+	               [regionJobRetryer]<-------------[dispatchJobGoroutine]-->done
 	*/
 
+	// Above is the happy path workflow of region jobs. A job is generated by
```

Comment:
maybe put all those comment at method level, so i can see it when cursor hover

Reply:
These are the inner logic of `doImport`. Do you think it's also helpful to see them at the caller of `doImport`? If you can confirm it I will move them.